kong/plugins/hmac-auth/access.lua

```diff
@@ -82,11 +87,26 @@ local function create_hash(request, hmac_params, headers)
   return ngx_sha1(hmac_params.secret, signing_string)
 end
 
+local function is_signature_equal(signature_1, signature_2)
```

Comment:
Sorry, I mentioned the wrong commit. I should have typed 6ccdf7b664957edef3f557c53ae706456c411e0d :)